core-web/libs/dotcms-scss/angular/dotcms-theme/components/form/_autocomplete.scss

```diff
@@ -35,14 +35,15 @@ p-autocomplete.p-inputwrapper-focus {
         grid-column-start: 2;
     }
 
-    .p-autocomplete-input {
+    .p-autocomplete-input.p-component {
         all: unset;
         border: none;
         font-family: $font-default;
         font-size: $font-size-md;
         color: $black;
         padding: 0;
         margin: 0;
+        width: 100%;
```

Comment:
Whattttt, this could mess up other things no?

Reply:
I checked and it looks like it doesn't break anything

Reply:
Cool.